### PR TITLE
Change style of user messages

### PR DESCRIPTION
**Pull request by bond95**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/119

### CODE REVIEW

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

**Comment by ovirt-infra** [comment]
The commiter is not a member of organization nor whitelist. Please contact maintainer for patch verification.

File: src/components/VmUserMessages/style.css [hunk 1]
```diff
@@ -20,9 +20,21 @@
...
+
+pre.message-box {
+	white-space: pre-wrap;
+}
+
+.usermsg-time {
+	text-align: right;
+    border-bottom: solid 1px lightgray;
+    margin-bottom: 10px;
+}
```
Comment (mareklibra) [inline src/components/VmUserMessages/style.css]: New line, please ...

Comment (bond95) [inline src/components/VmUserMessages/style.css]: Already added new line

**Pull request merged**